### PR TITLE
Null config fix

### PR DESCRIPTION
**Pull request by tjaved970**
added an if check to prevent crash

### CODE REVIEW

File: filestack/src/main/java/com/filestack/android/FsActivity.java [hunk 1]
```diff
@@ -166,20 +166,24 @@ protected void onCreate(Bundle savedInstanceState) {
...
+            String sessionToken = preferences.getString(PREF_SESSION_TOKEN, null);
+            Util.initializeClientIfNeeded(config, sessionToken);
+
+            if (savedInstanceState == null) {
+                Util.getSelectionSaver().clear();
+                selectedSource = sources.get(0);
+                nav.getMenu().performIdentifierAction(Util.getSourceIntId(selectedSource), 0);
+                if (drawer != null) {
+                    drawer.openDrawer(GravityCompat.START);
+                }
```
Comment (j1george) [inline filestack/src/main/java/com/filestack/android/FsActivity.java:179]: nit.
```suggestion
                drawer?.openDrawer(GravityCompat.START);
```

**Review by j1george — APPROVED** [review]

File: filestack/src/main/java/com/filestack/android/FsActivity.java [hunk 2]
```diff
@@ -166,20 +166,24 @@ protected void onCreate(Bundle savedInstanceState) {
...
+            } else {
+                selectedSource = savedInstanceState.getString(STATE_SELECTED_SOURCE);
+                shouldCheckAuth = savedInstanceState.getBoolean(STATE_SHOULD_CHECK_AUTH);
             }
         } else {
-            selectedSource = savedInstanceState.getString(STATE_SELECTED_SOURCE);
-            shouldCheckAuth = savedInstanceState.getBoolean(STATE_SHOULD_CHECK_AUTH);
+            setResult(FsConstants.RESULT_FILE_STACK_ERROR);
```
Comment (qtehfw) [inline filestack/src/main/java/com/filestack/android/FsActivity.java:185]: Are we going to handle this in the Fieldwire app?

**Review by qtehfw — APPROVED** [review]

File: filestack/src/main/java/com/filestack/android/FsActivity.java (hunk 2 above)
Comment (tjaved970) [reply filestack/src/main/java/com/filestack/android/FsActivity.java:185]: yup

File: filestack/src/main/java/com/filestack/android/FsActivity.java (hunk 1 above)
Comment (tjaved970) [reply filestack/src/main/java/com/filestack/android/FsActivity.java:179]: java class 😁

Comment (j1george) [reply filestack/src/main/java/com/filestack/android/FsActivity.java:179]: oh i didn't even notice that lol

Comment (tjaved970) [reply filestack/src/main/java/com/filestack/android/FsActivity.java:179]: haha :D

**Pull request merged**